lib/handler/access_log.c

```diff
@@ -24,7 +24,9 @@
 #include <inttypes.h>
 #include <netdb.h>
 #include <netinet/in.h>
+#ifndef __ANDROID__
 #include <spawn.h>
+#endif
```

Comment:
Could you please remove `#include <spawn.h>` as a whole and see what happens?

I believe that this is no longer needed; it was necessary when we called `posix_spawnp` directly, but now we use `h2o_spawnp`.